.github/ISSUE_TEMPLATE/marketing-request.md

```diff
@@ -0,0 +1,53 @@
+---
+name: Marketing Request
+about: Get help getting a message out to our community through coordinated outreach no email, social media, and other public channels.
+title: 'REQUEST: New communication about <topic>'
+labels: area/contributor-comms, sig/contributor-experience
+assignees: '@jonasrosland'
+---
+<!--
+ATTENTION: the Upstream Marketing Team have classifications on how urgent and important a message to the community is [outlined here][multichannel].
+
+Filling out this form to the best of your ability will speed up the team's ability to get the word out! 
+-->
+
+**What do you want send out?** 
+<!-- Please include a link whenever possible, logos, artwork, pics, diagrams, or any other assets with public links. -->
+
+
+**What’s the intended outcome?**
+<!-- Are you looking for people to get involved? Fill out a survey? See something of interest to them? Let us know why you care to share. -->
+
+
+**What is the sponsoring SIG or community group?**
+<!-- -->
+
+**Is this time-sensitive?** 
+<!-- You can delete or skip this if it's not relevant -->
+
+- [ ] Yes, it is **due by** <$DATE> <$TIME>
+- [ ] Yes, it **must be sent** <$DATE> <$TIME>
+- [ ] No, but I'd prefer <$DATE> <$TIME>
+
+**Where do you want it to be visible?**
+<!-- Note that email via our mailinst list is the default choice for all messages. See our [multichannel communication strategy][multichannel] for more details. Note that blogs and social media must meet our guidelines available [here](https://github.com/kubernetes/community/tree/master/communication/marketing-team#purpose). -->
```

Comment:
```suggestion
<!-- Note that email via our mailing list is the default choice for all messages. See our [multichannel communication strategy][multichannel] for more details. Note that blogs and social media must meet our guidelines available [here](https://github.com/kubernetes/community/tree/master/communication/marketing-team#purpose). -->
```
Like the earlier link this one won't appear as a markdown link and needs to be text =/

Maybe something like:
```
Note that email via our mailing list is the default choice for all messages. See our
multichannel communication strategy for more details. Note that blogs and social
media must meet our guidelines.

Multichannel communication strategy: http://git.k8s.io/community/communication/marketing-team/multichannel-communications.md#mapping-type-to-tier

Blog and Social Media Guidelines: https://git.k8s.io/community/communication/marketing-team#purpose
```

Reply:
Great point!